toxav/bwcontroller.c

```diff
@@ -186,6 +184,49 @@ static int on_update(BWController *bwc, const struct BWCMessage *msg)
     return 0;
 }
 
+static void bwc_handle_data(Tox *tox, uint32_t friendnumber, const uint8_t *data, size_t length, void *dummy)
+{
+    if (length - 1 != sizeof(struct BWCMessage)) {
+        return;
+    }
+
+    /* get BWController object from Tox and friend number */
+    ToxAV *toxav = (ToxAV *)tox_get_av_object(tox);
+
+    if (!toxav) {
+        return;
+    }
+
+    void *call = NULL;
```

Comment:
Why is this a void pointer? I guess once travis is fixed, I'll know the answer.

Reply:
i dont know. but changed now